debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleSnapshotChangeEventSource.java

```diff
@@ -186,6 +186,15 @@ private boolean areSameTimestamp(Long scn1, long scn2) throws SQLException {
 
             return Optional.of(rs.getLong(1));
         }
+        catch (SQLException e) {
+            if (e.getMessage().contains("no snapshot found based on specified time")) {
+                // DBZ-1446 In this use case we actually do not want to propagate the exception but
+                // rather return an empty optional value allowing the current SCN to take prior.
+                LOGGER.warn("No latest table SCN could be resolved, defaulting to current SCN");
```

Comment:
Not sure it is enough important to be a warning, given that can be a common and expected case and that in such case it is safe to trust the current SCN.

Reply:
That's a good point, I'll change it to `debug` instead since it really doesn't have any major impact in normal operation.

Reply:
@Naros I've elevated the level to info. The message happens only once and it can be useful information for issue diagnostics in case there are some unlanned sideeffects.